plugins/agents/ping/Ping.go

```diff
@@ -0,0 +1,113 @@
+package ping
+
+import (
+	"strings"
+	"time"
+
+	"github.com/abrander/agento/plugins"
+	"github.com/abrander/agento/timeseries"
+	gping "github.com/gansoi/gansoi/plugins/agents/ping"
+)
+
+type Data struct {
+	Loss    int    `json:"loss"`
+	Sent    int    `json:"sent"`
+	Replies int    `json:"replies"`
+	TimeAvg int64  `json:"timeavg"`
+	TimeMin int64  `json:"timemin"`
+	TimeMax int64  `json:"timemax"`
+	Ip      string `json:"server"`
+}
+
+type Ping struct {
+	Data   []Data `json:"data"`
+	pinger *gping.ICMPService
```

Comment:
Can be make `pinger` package level and initialize in `init()`?

Reply:
@abrander done